recipes/shorah/1.99.0/meta.yaml

```diff
@@ -32,6 +32,8 @@ requirements:
     - numpy
     - gsl
     - zlib
+    - setuptools
```

Comment:
setuptools _should_ suffice for now.

Reply:
OK, go ahead and remove pip and I'll approve the PR so it can get merged.

Reply:
Thanks!